packages/fluent/scss/listview/_layout.scss

```diff
@@ -0,0 +1,143 @@
+@use "../core/mixins" as *;
+@use "_variables.scss" as *;
+
+@mixin kendo-listview--layout() {
+
+    // Listview
+    .k-listview {
+        margin: 0;
+        padding: 0;
+        border-width: 0;
+        border-style: solid;
+        box-sizing: border-box;
+        outline: 0;
+        font-family: var( --kendo-listview-font-family, $kendo-listview-font-family );
+        font-size: var( --kendo-listview-font-size, $kendo-listview-font-size );
+        line-height: var( --kendo-listview-line-height, $kendo-listview-line-height );
```

Comment:
Interpolate sass variables (  see [the official docs](https://sass-lang.com/documentation/breaking-changes/css-vars) for ref  ). Please do so for all fallback values where a sass variable is used in the entire file.